src/pages/overzicht/Overzicht.jsx

```diff
@@ -0,0 +1,45 @@
+import './Overzicht.css';
+import posts from '../../constants/data.json';
+import formatDate from '../../helpers/formatDate.js';
+import {Link} from "react-router-dom";
+import Navigation from '../../components/navigation/Navigation';
+
+
+function Overzicht() {
+
+    return (
+        <>
+            <header><h1>Overzicht</h1></header>
+            <main>
+                <div className="outer-container">
+                    <h1>Overzicht Pagina</h1>
```

Comment:
in de h1 mist nog een nummeriek getal van het aantal post kan je oplossen met .lenght